salt/modules/win_lgpo.py

```diff
@@ -3439,7 +3439,7 @@ def _processValueItem(element, reg_key, reg_valuename, policy, parent_element,
             element_valuenames = []
             element_values = this_element_value
             if this_element_value is not None:
-                element_valuenames = list(range(1, len(this_element_value) + 1))
+                element_valuenames = list([str(z) for z in range(1, len(this_element_value) + 1)])
```

Comment:
Was this tested on PY3 by chance, or just PY2? I am wondering if this usage of `str` will work correctly on both versions.

Reply:
I will go back and test both again, just to make sure

there are almost 20 calls to the str() function in places throughout the module

Reply:
re-validated that it works on both PY2 and PY3

Reply:
OK thanks @lomeroe for checking that!